go/pod/init.go

```diff
@@ -0,0 +1,6 @@
+package pod
+
+func Init() error {
+	_, err := getOrDownloadPodWorker()
```

Comment:
I would download here, and skip if it's only there. could log to DEBUG (downloading, download status, or skipping) or something.